scheduler/scheduler.py

```diff
@@ -6,6 +6,7 @@ class Scheduler():
     def __init__(self):
         self.sched = BackgroundScheduler(timezone='EST')
         self.jobSet = []
+        self.sched.start()
         logging.basicConfig(filename='scheduler.log',level=logging.DEBUG)
```

Comment:
It seems like scheduling jobs works, but you never grab any values or send it to the function. You should write a wrapper function for the scheduled task that grabs the newest values and passes them as arguments.
```
def task_wrapper(actual_func):
    patient_id, bps, bpd, pulse, oxygen = data_vendor.get_values()
    actual_func(patient_id, bps=bps, bpd=bpd, pulse=pulse, oxygen=oxygen)

```

Reply:
It doesn't have to be exactly this, but something similar should work pretty well.

Reply:
added it